extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,129 @@
+# Tiled Assets specification
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
```

Comment:
Could you align this header section to how we have it now in dev for all other extension, please? With Field Name Prefix etc?

Reply:
👍